query-node/schema.graphql

```diff
@@ -83,23 +201,25 @@ type Channel @entity {
   # or a member. We are not handling events related to curator group so we will not set this field
   # owner: Member!
 
+  category: ChannelCategory
+
   "The title of the Channel"
-  handle: String! @fulltext(query: "search")
+  title: String @fulltext(query: "search")
 
   "The description of a Channel"
-  description: String!
+  description: String
 
   "Url for Channel's cover (background) photo. Recommended ratio: 16:9."
-  coverPhotoUrl: String
+  coverPhotoUrl: Asset
```

Comment:
We will need to use one common naming for the assets, `Url` suffix will not work anymore. I guess it could be just `coverPhoto`?

Reply:
Yes, I made the same point on another field also (thumbnailUrl -> thumbnail)

Reply:
Done